src/EFCore.PG/Extensions/NpgsqlEntityTypeBuilderExtensions.cs

```diff
@@ -136,5 +140,39 @@ public static class NpgsqlEntityTypeBuilderExtensions
             => (EntityTypeBuilder<TEntity>)ForCockroachDbInterleaveInParent((EntityTypeBuilder)entityTypeBuilder, parentTableType, interleavePrefix);
 
         #endregion CockroachDB Interleave-in-parent
+
+        #region Generic Index
+
+        /// <summary>
+        ///     Configures an index on the specified properties. If there is an existing index on the given
+        ///     set of properties, then the existing index will be returned for configuration.
+        /// </summary>
+        /// <typeparam name="TEntity"> The entity type being configured. </typeparam>
+        /// <param name="entityTypeBuilder"> The builder for the entity type being configured. </param>
+        /// <param name="indexExpression">
+        ///     <para>
+        ///         A lambda expression representing the property(s) to be included in the index
+        ///         (<c>blog => blog.Url</c>).
+        ///     </para>
+        ///     <para>
+        ///         If the index is made up of multiple properties then specify an anonymous type including the
+        ///         properties (<c>post => new { post.Title, post.BlogId }</c>).
+        ///     </para>
+        /// </param>
+        /// <returns> An object that can be used to configure the index. </returns>
```

Comment:
nit: Omit extra spaces in `returns` node.

Reply:
This is ALL OVER the code base already. Should I really do this as part of this PR?

Reply:
Sigh... I thought we took care of these a few months back in a solution-wide cleanup PR, but they must still be sitting in a local branch.

If you're seeing them in other files, no need to worry about it here.

Reply:
I just did a quick search and found 180 occurrences across 20 files, so it would probably be better to handle them as a separate PR 😄

Reply:
Yikes! Yes, best to leave these nits for another time.